zebra/rib.h

```diff
@@ -146,6 +140,12 @@ struct route_entry {
 
 	/* Sequence value incremented for each dataplane operation */
 	uint32_t dplane_sequence;
+
+	/* Source protocol instance */
+	uint8_t instance;
```

Comment:
shouldn't `instance` be 16-bit wide? The `router ospf` command accepts `(1-65535)` as the instance parameter.

Reply:
@donaldsharp

Reply:
Yes it should.